<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Positioning</class>
 <widget class="QWidget" name="Positioning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="b_obj2_e">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Icons/Arrows/right218.png</normaloff>../../../../Control/ControlInterface/Icons/Arrows/right218.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_e">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Tango/go-next2.png</normaloff>../../../../Control/ControlInterface/Tango/go-next2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sample_ref">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_obj_amp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_np_steps">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_up">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Icons/Arrows/ascendant.png</normaloff>../../../../Control/ControlInterface/Icons/Arrows/ascendant.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_step1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_se">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_xy_ref">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>270</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_s">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Icons/Arrows/down14.png</normaloff>../../../../Control/ControlInterface/Icons/Arrows/down14.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_nw">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_nw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_step2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_z_ref">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_step4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_ne">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj1_z_ref">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_obj_steps">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Steps</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>20</width>
     <height>121</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_step5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_s">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Tango/go-down2.png</normaloff>../../../../Control/ControlInterface/Tango/go-down2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_step5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_down">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Tango/go-down2.png</normaloff>../../../../Control/ControlInterface/Tango/go-down2.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_np_obj1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Obj 1 (Z)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_up">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Tango/go-up2.png</normaloff>../../../../Control/ControlInterface/Tango/go-up2.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_np_obj1_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Obj 2 (Z)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_step3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ref">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_ne">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_sw">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>20</width>
     <height>221</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_step6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>5000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_obj_steps">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_obj2_z_alt">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_n">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Tango/go-up2.png</normaloff>../../../../Control/ControlInterface/Tango/go-up2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_step1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_np_sample">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample (XY)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_w">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Icons/Arrows/left207.png</normaloff>../../../../Control/ControlInterface/Icons/Arrows/left207.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_down">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Icons/Arrows/down14.png</normaloff>../../../../Control/ControlInterface/Icons/Arrows/down14.png</iconset>
   </property>
  </widget>
  <widget class="QCheckBox" name="c_obj1_z_alt">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_obj_amp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_step4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_w">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Tango/go-previous2.png</normaloff>../../../../Control/ControlInterface/Tango/go-previous2.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_np_obj2_xy">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Obj 2 (XY)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_step2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_se">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_sw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="b_np_step6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>2000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_step3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_np_steps">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_obj2_n">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../../../Control/ControlInterface/Icons/Arrows/ascendant.png</normaloff>../../../../Control/ControlInterface/Icons/Arrows/ascendant.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="b_init">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Initialise</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
